docs/pages/access-controls/sso/okta.mdx

```diff
@@ -137,37 +163,39 @@ spec:
 ```
 
 - Devs are only allowed to login to nodes labelled with `access: relaxed` label.
-- Developers can log in as `ubuntu` user
-- Developers also do not have any "allow rules" i.e. they will not be able to
+- Developers can log in as `ubuntu` user or a user matching their email prefix.
+- Developers do not have any "allow rules" i.e. they will not be able to
   see/replay past sessions or re-configure the Teleport cluster.
 
-We'll use tctl to create this role on the auth server:
+Use tctl to create this role on the auth server:
```

Comment:
```suggestion
Use tctl to create this role on the Auth Service:
```

Reply:
Also, does this need to be on the Auth Service, or can we use tctl remotely here?

Reply:
This is saying that the role exists in the Auth Service after creation, not that `tctl` is run on the Auth Service host specifically. I'll see if I can rephrase.